ante/sigverify.go

```diff
@@ -45,6 +46,10 @@ func SigVerificationGasConsumer(
 		meter.ConsumeGas(secp256k1VerifyCost, "ante verify: eth_secp256k1")
 		return nil
 
+	case *ethermintethsecp256k1.PubKey:
```

Comment:
Should we continue to use the same two key types?
Perhaps existing wallets including keyrings are storing ethermint types, and are you considering supporting both without migration?